rust/worker/src/compactor/compaction_manager.rs

```diff
@@ -113,7 +113,7 @@ impl CompactionManager {
                     dispatcher.clone(),
                     None,
                     None,
-                    Arc::new(AtomicU32::new(1)),
```

Comment:
is this correct?

Reply:
Yes. This is the default value that we are passing in case the segment is uninitialized. Previously we were passing the next_offset_id (as opposed to curr_max_offset_id). I changed that to pass curr_max_offset_id now.